python/dlr/api.py

```diff
@@ -50,7 +50,7 @@ def _is_module_found(name):
 
 # Wrapper class
 class DLRModel(IDLRModel):
-    def __init__(self, model_path, dev_type=None, dev_id=None, error_log_file=None):
+    def __init__(self, model_path, dev_type=None, dev_id=None, error_log_file=None, use_default_dlr=False):
```

Comment:
```
class DLRModelImpl(IDLRModel):
    def __init__(self, model_path, dev_type='cpu', dev_id=0, use_default_dlr=False, logger=None):
```

Reply:
logger is not same as error_log_file

Reply:
Lets use the same name for parameters and make sure that `error_log_file` is before `use_default_dlr`

Reply:
Do we want to pass the error_log_file, Why instantiate the logger multiple times? Its better to pass the logger with the right name we should avoid confusing names. I can fix the order

Reply:
Looking at it again, it is better to keep it consistent. 👍